sqlobject/src/main/java/org/jdbi/v3/sqlobject/statement/internal/ResultReturner.java

```diff
@@ -109,7 +109,11 @@ private static Object checkResult(Object result, QualifiedType<?> type) {
         return result;
     }
 
-    static class VoidReturner extends ResultReturner {
+    public static ResultReturner forVoid() {
```

Comment:
what's with the changes in this file? Doesn't seem related, this method for example isn't called in this diff.

Reply:
Yeah, this was part of a refactoring that turned out not to be necessary.